e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc.yml

```diff
@@ -63,6 +101,8 @@ spec:
               value: localhost
             - name: DB_PORT
               value: "3306"
+            - name: PMM_AGENT_PRERUN_SCRIPT
+              value: pmm-admin add $(DB_TYPE) --server-url=https://$(PMM_AGENT_SERVER_USERNAME):$(PMM_AGENT_SERVER_PASSWORD)@$(PMM_AGENT_SERVER_ADDRESS)/  --skip-connection-check  --server-insecure-tls  --username=$(DB_USER) --password=$(DB_PASSWORD) --cluster=$(DB_CLUSTER) $(PMM_AGENT_SETUP_NODE_NAME) $(DB_HOST):$(DB_PORT) $(DB_ARGS)
```

Comment:
* `--server-url`, `--server-insecure-tls` can be removed;
* please use `--service-name=$(PMM_AGENT_SETUP_NODE_NAME)`;
* please use `--host=$(DB_HOST) --port=$(DB_PORT)`.

Reply:
thank you